test/vegur_proxy_SUITE.erl

```diff
@@ -68,19 +72,28 @@ init_per_suite(Config) ->
     {ok, Cowboyku} = application:ensure_all_started(cowboyku),
     {ok, Inets} = application:ensure_all_started(inets),
     {ok, Meck} = application:ensure_all_started(meck),
-    VegurPort = 9333,
-    {ok, _} = vegur:start_http(VegurPort, vegur_stub, []),
-    mock_through(),
-    [{started, Cowboyku++Inets++Meck},
-     {vegur_port, VegurPort} | Config].
+    [{started, Cowboyku++Inets++Meck}|Config].
 
 end_per_suite(Config) ->
-    vegur:stop_http(),
     application:unload(vegur),
     application:unload(websocket_client),
     [application:stop(App) || App <- ?config(started, Config)],
     Config.
 
+init_per_group(http, Config) ->
+    VegurPort = 9333,
+    {ok, _} = vegur:start_http(VegurPort, vegur_stub, []),
+    mock_through(),
+    [{vegur_port, VegurPort}, {scheme, "http"} | Config];
+init_per_group(https, Config) ->
+    VegurPort = 9333,
+    SSLOptions = [{certfile,   "../../../../test/localhost.crt"},
+                  {keyfile,    "../../../../test/localhost.key"},
```

Comment:
I'd just copy the _data dir for many suites in this case. I'd rather have the data dir working since that makes for more portable test runs